src/containerapp/azext_containerapp/_transformers.py

```diff
@@ -3,7 +3,7 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
-
+from azure.cli.command_modules.containerapp._utils import safe_get
```

Comment:
The **Azure Core CLI** (aka. [Azure CLI](https://learn.microsoft.com/en-us/cli/azure/what-is-azure-cli)) contains the key features for different Azure Resource provider. Starting from the 2.53.0 the Container App command has been added as a part of Azure the Core CLI
Currently the value of `azext.minCliCoreVersion` is not enough to import `safe_get`.
After [this PR](https://github.com/Azure/azure-cli-extensions/pull/6868) merged, update `"azext.minCliCoreVersion"` to `"2.53.0"`, you can import from `azure.cli.command_modules.containerapp`.
```suggestion
from ._utils import safe_get
```